solution/backend/regulations/templates/regulations/partials/header.html

```diff
@@ -1,20 +1,23 @@
 <header id="header" class="sticky">
     <header-component home-url="{% url 'homepage' %}">
-        <jump-to
-            slot="jump-to"
-            api-url="{{ API_BASE }}"
-            home-url="{% url 'homepage' %}"
-            {% if title %}title="{{ title }}"{% endif %}
-            {% if part %}part="{{ part }}"{% endif %}
-        ></jump-to>
-        <header-links
-            slot="links"
-            statutes-url="{% url 'statutes' %}"
-            subjects-url="{% url 'subjects' %}"
-        ></header-links>
-        <header-search
-            slot="search"
-            search-url="{% url 'search' %}"
-        ></header-search>
+        <template #jump-to>
+            <jump-to
+                api-url="{{ API_BASE }}"
+                home-url="{% url 'homepage' %}"
+                {% if title %}title="{{ title }}"{% endif %}
+                {% if part %}part="{{ part }}"{% endif %}
+            ></jump-to>
+        </template>
+        <template #links>
+            <header-links
+                statutes-url="{% url 'statutes' %}"
+                subjects-url="{% url 'subjects' %}"
+            ></header-links>
+        </template>
+        <template #search>
+            <header-search
+                search-url="{% url 'search' %}"
+            ></header-search>
+        </template>
```

Comment:
Named slots must be used on a <template> element. See https://vuejs.org/guide/components/slots.html#named-slots